app/models/stack.js

```diff
@@ -28,10 +28,7 @@ export default DS.Model.extend({
 
   // computed properties
   allowPHI: Ember.computed.match('type', /production/),
-  appContainerCentsPerHour: Ember.computed('allowPHI', function() {
-    return this.get('allowPHI') ? 10 : 6;
-  }),
-
+  appContainerCentsPerHour: '8',
```

Comment:
If this value is used in math (e.g. calculating expected total values for a given container @ X hours based on the cents-per-hour value) it should be a number instead of a string.